src/app/features/collectibles/components/collectible-item.layout.tsx

```diff
@@ -60,9 +61,14 @@ export function CollectibleItemLayout({
             left="0px"
             overflow="hidden"
             position="absolute"
+            style={{
```

Comment:
Maybe we can write a short function for this?

Double ternary can be hard to read and could even be banned by eslint

It works though!